doc/COMPILING/COMPILING-MSYS.md

```diff
@@ -1,48 +1,50 @@
-# Compilation guide for 64 bit Windows (using MSYS2)
+# Compilation guide for 64-bit Windows (using MSYS2)
 
-This guide contains steps required to allow compilation of Cataclysm-DDA on Windows under MSYS2.
+This guide contains instructions for compiling Cataclysm-DDA on Windows under MSYS2. **PLEASE NOTE:** These instructions *are not intended* to produce a redistributable copy of CDDA. Please download the official builds from the website or [cross-compile from Linux](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/COMPILING/COMPILING.md#cross-compile-to-windows-from-linux) if that is your intention.
 
-Steps from current guide were tested on Windows 10 (64 bit) and MSYS2 (64 bit), but should work for other versions of Windows and also MSYS2 (32 bit) if you download 32 bit version of all files.
+These instructions were written using 64-bit Windows 7 and the 64-bit version of MSYS2; the steps should be the same for other versions of Windows.
 
 ## Prerequisites:
 
-* Computer with 64 bit version of modern Windows operating system installed (Windows 10, Windows 8.1 or Windows 7);
-* NTFS partition with ~10 Gb free space (~2 Gb for MSYS2 installation, ~3 Gb for repository and ~5 Gb for ccache);
-* 64 bit version of MSYS2 (installer can be downloaded from [MSYS2 homepage](http://www.msys2.org/));
+* Windows 7, 8, 8.1, or 10
+* NTFS partition with ~10 Gb free space (~2 Gb for MSYS2 installation, ~3 Gb for repository and ~5 Gb for ccache)
+* 64-bit version of MSYS2
 
 **Note:** Windows XP is unsupported!
 
 ## Installation:
 
-1. Go to [MSYS2 homepage](http://www.msys2.org/) and download 64 bit installer (e.g. [msys2-x86_64-20180531.exe](http://repo.msys2.org/distrib/x86_64/msys2-x86_64-20180531.exe)).
+1. Go to the [MSYS2 homepage](http://www.msys2.org/) and download the installer.
 
-2. Run downloaded file and install MSYS2 (click `Next` button, specify directory where MSYS2 64 bit will be installed (e.g. `C:\msys64`), click `Next` button again, specify Start Menu folder name and click `Install` button).
+2. Run the installer. It is suggested that you install to a dev-specific folder (C:\dev\msys64\ or similar), but it's not strictly necessary.
 
-3. After MSYS2 installation is complete press `Next` button, tick `Run MSYS2 64 bit now` checkbox and press `Finish` button.
+3. After installation, run MSYS2 64bit now.
 
 ## Configuration:
 
-1. Update the package database and core system packages with:
+1. Update the package database and core system packages:
 
 ```bash
-pacman -Syu
+pacman -Syyu
 ```
 
-2. If asked close MSYS2 window and restart it from Start Menu or `C:\msys64\msys2_shell.cmd`.
+2. MSYS will inform you of a cygheap base mismatch and inform you a forked process died unexpectedly; these errors appear to be due to the nature of `pacman`'s upgrades and *may be safely ignored.* You will be prompted to close the terminal window; do so, then re-start using the MSYS2 MinGW 64-bit menu item.
```

Comment:
Different versions of third party tools that can report different errors at different time. These errors aren't CDDA-specific and should not be addressed in CDDA repo.

Reply:
They're not CDDA-specific, no, but they *are* MSYS2-specific, and Eso told me I should call it out.

> 4:40 PM] esotericist: if it turns out the error is due to your step, but is harmless, you should still document it because odds are good someone else might do that innocently

The error, as it turns out, is not due to a step I took, but will always happen when you follow the directions we give, and so it should be noted.

Reply:
It is noted on MSYS2 site iirc.

Reply:
Even if something isn't CDDA-specific, mentioning said errors may help divert people from complaining about them in CDDA fora (issues, discord...).

Reply:
> It is noted on MSYS2 site iirc.

Sure, but users would only be on MSYS2's website to download the installer. They're following our instructions for compilation, and as NH says, it's our due diligence.